index.html

```diff
@@ -229,8 +263,10 @@ <h4 class="about-card-title">Paweł Racki</h4>
                         </div>
                         <div class="about-card-body">
                             <h4 class="about-card-title font-normal">Patryk Fijałkowski</h4>
-                            <p>Polish Onion - nigdy nie kupuję w regularnych cenach</p>
-                            <p>Więcej o mnie: </p>
+                            <p data-lang="pl" style="display: block;">Polish Onion - nigdy nie kupuję w regularnych cenach</p>
+                            <p data-lang="en" style="display: none;">Polish Onion - regular prices are unknown to me</p>
+                            <p data-lang="pl" style="display: block;">Więcej o mnie: </p>
+                            <p data-lang="en" style="display: none;">More about me: </p>
```

Comment:
...a raz jako 'en'...
nie wiem czy to ma jakiekolwiek znaczenie.
Pewno działa, ale zaznaczam bo po prostu zwróciło moją uwagę :)